test/__fixtures/BaseTest.sol

```diff
@@ -122,6 +122,7 @@ abstract contract BaseTest is Test {
         int56 aLiq,
         uint32 aTime
     ) internal {
+        require(aTime < 2 ** 31, "TIMESTAMP TOO BIG");
```

Comment:
technically this can be `aTime <= 2 ** 31` right?

Reply:
Hmm, I think not right, because the max value for uint31 is 2**31 - 1 right? (it's always -1 because of 0)? That said I was lucky, didnt think too hard about this

Reply:
I think you're right, the max value should be 2 ** 31 - 1, so your require statement is fine